src/ex4/server/services/storage.js

```diff
@@ -0,0 +1,72 @@
+import Item from "../db/models/item.js";
+import { sequelize } from "../db/models/index.js";
+
+class StorageService {
```

Comment:
would suggest to rename it to `TasksModelService` which indicates that this service is using DB model as `StorageService` is meaningless